pkg/controller/remotemachineset/gcpactuator.go

```diff
@@ -451,6 +463,32 @@ func getGCPImageID(masterMachine *machineapi.Machine, scheme *runtime.Scheme, lo
 	return imageID, nil
 }
 
+// Get the network information from existing machines on the remote cluster.
+func getNetwork(remoteMachineSets []machineapi.MachineSet,
```

Comment:
let's make sure we add unit tests for this function.

Reply:
Sure.